### DISCUSSION TITLE
230705 git branch 전략 추가 회의 (11:50 ~ 12:05)

### DISCUSSION THREAD

**Discussion by Creative-Lee** [woowacourse-teams/2023-shook#21] [회의]
## 회의 안건

![image](https://github.com/woowacourse-teams/2023-shook/assets/77152650/fdae51b4-18b4-4fa7-9019-df0158fb241b)
![image](https://github.com/woowacourse-teams/2023-shook/assets/77152650/ce187290-4341-4c99-8ade-a91d4ea3eb7d)

hotfix 브랜치에 대해 회의한다.

**Comment by somsom13** [comment]
### Branch 전략 추가 회의
- 우리가 알던 Git Flow 흐름으로 이해하는 방향
- 이해 관점의 차이였음
